lab-taylor/route/team-router.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
+
+const debug = require('debug')('authlab:team-router');
+const Router = require('express').Router;
+const httpErrors = require('http-errors');
+const teamController = require('../controller/team-controller');
+const parseBearerAuth = require('../lib/parse-bearer-auth');
+
+
+const teamRouter = module.exports = new Router();
+const jsonParser = require('body-parser').json();
+
+teamRouter.post('/team', parseBearerAuth, jsonParser, (req, res, next) => {
```

Comment:
This router could be cleaned up by using the .route method:

teamRouter.route('/team')
.all( commonMiddleware, moreCommonMiddleware )
.post( specificMiddleware, (req, res, next) => // code )
.get( specificMiddleware, (req, res, next) => // code );

then do the same for teamRouter.route('/team/:id')